Sources/Badgy/main.swift

```diff
@@ -16,4 +16,8 @@ cli.commands = [
 cli.globalOptions.append(ReplaceFlag)
 cli.globalOptions.append(VerboseFlag)
 
+#if ARGUMENT_PARSER
+Badgy.main()
+#else
 _ = cli.go()
```

Comment:
Would this even work now?

Reply:
Yeap, both CLIs are functional, `SwiftCLI` is the default one.

You can active `ArgumentParser` version with:

```bash
swift run -Xswiftc -DARGUMENT_PARSER badgy
```